.gitignore

```diff
@@ -1,5 +1,5 @@
 .idea
 *.iml
 target
-Cargo.lock
```

Comment:
Please do not remove Cargo.lock. It is necessary for libraries to keep it out of tree.